src/index.tsx

```diff
@@ -103,7 +114,7 @@ export function PackedGrid({
         placeContent: 'center'
       }}
     >
-      {React.Children.map(children, (child) => (
+      {Children.map(children, (child) => (
         <div
```

Comment:
I think this would be simpler as:

```jsx
{React.Children.map(children, (child) => (
        {child && <div
```

Would this allow us to remove the pre-filtering / renderToStaticMarkup above?

Reply:
I thought filtering children up-front was cleaner. Shouldn't `numBoxes` be synced with the number of wrappers rendered?

Reply:
If you only think the lib should filter out null-ish children, the initial 
```js
children = React.Children.toArray(children).filter(Boolean)
```
should do it. Right? 

Should I revert the PR to that commit?

Reply:
ahh yeah, `numBoxes` does indeed need to be synced. I think the original commit looks correct - lets revert to that.

thanks!